plugins/action/jcliff.py

```diff
@@ -7,9 +7,9 @@
     file to the target host. """
 import os
 import tempfile
+import json
 
 from ansible.plugins.action import ActionBase
-from ansible.template import Templar
 from ansible.utils.display import Display
 
 display = Display()
```

Comment:
Unrelated to the PR, but you can avoid this, see: https://github.com/ansible-collections/amazon.aws/commit/ab3cea96e256b884b9f1cac4dde5dc1a5fea0675